### PR TITLE
docs: new example to highlight multiline behavior

### PR DESCRIPTION
**Pull request by robszumski**
fixes https://github.com/coreos/fleet/issues/1511

cc: @kayrus @daniellowtw

### CODE REVIEW

File: Documentation/unit-files-and-scheduling.md [hunk 1]
```diff
@@ -141,6 +141,20 @@ the logic would be as follows:
 diskType=SSD AND (region=us-east-1 OR region=us-west-1)
 ```
 
+Grouping metadata pairs onto separate lines has no affect on the matching logic:
+
+```sql
```
Comment (kayrus) [inline Documentation/unit-files-and-scheduling.md]: s/sql/ini/

Comment (robszumski) [inline Documentation/unit-files-and-scheduling.md]: the rest of them have this, was just copying

Comment (robszumski) [inline Documentation/unit-files-and-scheduling.md]: oh nevermind, i see what you're saying :)

**Comment by kayrus** [comment]
LGTM, @jonboulle, your opinion?

**Pull request merged**